convert2rhel/main.py

```diff
@@ -56,6 +56,9 @@ def main():
 
         process_phase = ConversionPhase.POST_CLI
 
+        # the tool will not run if the system not executed in single-user mode
+        utils.check_if_single_user_mode()
+
```

Comment:
Call this after `rhelvariant.determine_rhel_variant()` so we know whether the system can be converted or not. Otherwise they might boot into single-user mode, and not accept EULA or simply not be on a compatible system